channels/microsoft365-graph/accounts.rst

```diff
@@ -0,0 +1,126 @@
+Accounts
+========
+
+You can connect to user and shared mailboxes in your M365 environment.
+Follow the steps below and skip not applicable parts.
+
+First Steps
+-----------
+
+- Check your FQDN under *Settings > System > Fully Qualified Domain Name* in the
+  admin interface of Zammad. If it is not correct, change it now. Otherwise the
+  setup of the channel will fail.
+- Go to *Channels > Microsoft 365 Graph Email* and click on
+  **Connect Microsoft 365 App**. Copy the provided callback URL.
+
+Configuration
+----------------
+
+Go to Microsoft's `Entra admin center <https://entra.microsoft.com/#home>`_ and
+log in with as an administrator.
```

Comment:
"log in with as" sounds wrong? And also it needs not to be an administrator account only, you need only application administrator permission, but maybe only saying "administrator" is easier to understand.